underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -439,27 +559,8 @@ private String stripFolderSuffixIfPresent(String key) {
    * @param path the key to strip
    * @return the key without the Swift container prefix
    */
-  private String stripPrefixIfPresent(String path) {
-    if (path.startsWith(PATH_SEPARATOR)) {
-      return stripPrefixIfPresent(path, PATH_SEPARATOR);
-    }
-    return stripPrefixIfPresent(path, mContainerPrefix);
-  }
-
-  /**
-   * Strips the Swift container prefix from the key if it is present. For example, for input key
-   * swift://my-container-name/my-path/file, the output would be my-path/file. This method will
-   * leave keys without a prefix unaltered, ie. my-path/file returns my-path/file.
-   *
-   * @param path the key to strip
-   * @param prefix prefix to remove
-   * @return the key without the Swift container prefix
-   */
-  private String stripPrefixIfPresent(String path, String prefix) {
-    if (path.startsWith(prefix)) {
-      return path.substring(prefix.length());
-    }
-    return path;
+  private String stripPrefixIfPresent(final String path) {
```

Comment:
Why do some call `UnderFileSystemUtils.stripPrefixIfPresent` directly, and others call this `stripPrefixIfPresent`?